fastcrypto-zkp/src/bn254/poseidon/mod.rs

```diff
@@ -271,4 +301,25 @@ mod test {
         let inputs = vec![vec![255; 31]];
         assert!(hash_to_bytes(&inputs).is_ok());
     }
+
+    #[cfg(test)]
```

Comment:
are there any official test vectors or can we fuzzy test against another Poseidon lib (ie the previous one?)?

Reply:
There are [test vectors](https://extgit.iaik.tugraz.at/krypto/hadeshash/-/blob/master/code/test_vectors.txt?ref_type=heads) which we're aligned with, but I don't know any test vectors for edge cases.

I've added a proptest to verify alignment with poseidon-rs for 1-16 inputs, but it's not checking edge cases. I'll look into what options we have.